drake/systems/plants/@RigidBodyManipulator/doKinematics.m

```diff
@@ -99,6 +108,20 @@
   else
     kinsol.mex_ptr = options.kinematics_cache_ptr_to_use;
```

Comment:
There is only 'new kinsol' at this point. I should get rid of
```
if ~isfield(options, 'force_new_kinsol'), options.force_new_kinsol = false; end
```
which is the only remaining occurrence of `force_new_kinsol` in drake.